compat.h

```diff
@@ -14,8 +14,13 @@
 
 #include "access/tupdesc.h"
 #include "miscadmin.h"
-#include "storage/shm_mq.h"
-#include "utils/guc_tables.h"
+#include "storage/latch.h"
+
+#if PG_VERSION_NUM >= 110000
+typedef uint64 pgwsQueryId;
+#else
+typedef uint32 pgwsQueryId;
+#endif
```

Comment:
This adds additional space to profile hash key for older PG versions and as consequence increases the size of profile hash table. Also this segregation in type definition was added to pg_wait_sampling before - I just moved it to compat.h to refuse using of `#if` directive in code.

Reply:
Query.queryId is 64-bit since v11. I don't think we need to worry about allocating a bit more memory than strictly necessary for v10 and below. But surely we can leave this for another patch, if you like.